.github/workflows/docker.yaml

```diff
@@ -3,6 +3,7 @@
 name: Build and push docker images
 
 on:
+  pull_request:
```

Comment:
Can we only build the ARM images for tags and `main`? (since it seems to take ~an hour.) We should do something similar in Synapse's docker job.

Reply:
Oh, forgot to add: let's not push for PRs either, if we can avoid it.

But then Brad explained that this is only really here as a way to test that the changes do make a sha label, so maybe let's not worry!

Reply:
```suggestion
```